app/presenters/nfg_onboarder/high_level_navigation_bar_presenter.rb

```diff
@@ -20,25 +22,45 @@ def render_previous_button_unless?
     # for the step nav body text
     # or default to humanizing the
     # step's symbol.
-    def step_body(step)
-      I18n.t(step, scope: h.locale_namespace + [:step_navigations], default: step.to_s.humanize)
+    def step_body(nav_step)
+      I18n.t(
+        nav_step,
+        scope: h.locale_namespace + [:step_navigations],
+        default: nav_step.to_s.split('_').map(&:capitalize).join(' ')
```

Comment:
If you are trying to get each element capitalized, you can use `nav_step.to_s.humanize.titleize`